topology/University of Nebraska/Nebraska-Omaha/Swan.yaml

```
@@ -0,0 +1,50 @@
+GroupDescription: Swan Cluster
+GroupID: 388
```

```suggestion
GroupID: 1295
```

Bump resource group ID